Gemfile.lock

```diff
@@ -41,21 +41,20 @@ GEM
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.6.0)
     rspec-support (3.6.0)
-    thread_safe (0.3.6)
-    tzinfo (1.2.10)
-      thread_safe (~> 0.1)
+    tzinfo (2.0.5)
+      concurrent-ruby (~> 1.0)
     zeitwerk (2.6.6)
 
 PLATFORMS
-  ruby
+  arm64-darwin-21
```

Comment:
Do you know why this changed? Our other `Gemfile.lock` files just keep the `ruby` platform

https://github.com/wunderteam/portal/blob/d552788a741017d4924288fa6d435c3a2590d1c6/Gemfile.lock#L843-L844

Reply:
I'm not sure why it changed but I was able to use bundler to add the `ruby` platform and remove the `arm64-darwin-21` platform.

```
❯ bundle lock --add-platform ruby
Fetching gem metadata from https://rubygems.org/........
Resolving dependencies...
Writing lockfile to /Users/corasutton/Developer/wunder/nano-service/Gemfile.lock

❯ bundle lock --remove-platform arm64-darwin-21
Writing lockfile to /Users/corasutton/Developer/wunder/nano-service/Gemfile.lock
```